logging_format/tests/test_visitor.py

```diff
@@ -130,6 +130,59 @@ def test_extra_with_nont_whitelisted_keyword():
     assert_that(visitor.violations[0][1], is_(equal_to(WHITELIST_VIOLATION.format("hello"))))
 
 
+def test_debug_ok_with_not_whitelisted_keyword():
+    """
+    Extra keyword is ok for debug if not in whitelist.
+
+    """
+    tree = parse(dedent("""\
+        import logging
+
+        logging.debug(
+            "Hello {hello}!",
+            extra=dict(
+              hello="{}",
```

Comment:
I'd probably use a different kwarg here to ensure that the test is validating the debug (vs info) case.